rasa/cli/initial_project/tests/conversation_tests.md

```diff
@@ -0,0 +1,40 @@
+## happy path 1
```

Comment:
Let's add a comment which describes the file and links to the docs at the top.

Reply:
@wochinge just did that :) However, there is a dedicated issue for that as well.

Reply:
@alwx I meant to this file (`rasa/cli/initial_project/tests/conversation_tests.md`)

Reply:
@alwx

Reply:
it's included now (#happy path 2)

Reply:
@alwx Sorry for the unclear comments. I suggested adding a comment like "This file contains tests to evaluate that your bot behaves as expected. If you want to learn more, please see the docs <here>"

Reply:
oh :) ok then. sorry!

Reply:
No worries. It's my fault when I can express myself clearly :-D